lib/src/Base/Func/EvaluationImplementation.cxx

```diff
@@ -555,6 +555,108 @@ Graph EvaluationImplementation::draw(const Point & xMin,
   return draw(0, 1, 0, Point(2), xMin, xMax, pointNumber, scale);
 }
 
+/** Draw the cross-cuts of the function supposed to have 1D output */
+GridLayout EvaluationImplementation::drawCrossCuts(const Point& centralPoint,
+  const Point& xMin,
+  const Point& xMax,
+  const Indices& pointNumber,
+  const Bool withMonoDimensionalCuts,
+  const Bool isFilled,
+  const Scalar vMin,
+  const Scalar vMax) const
+{
+  const UnsignedInteger inputDimension = getInputDimension();
+  if(!(inputDimension >= 2)) throw InvalidArgumentException(HERE) << "Error: cannot draw cross cuts of a function with input dimension=" << inputDimension << " less than 2 using this method. See the other draw() methods.";
```

Comment:
Add a space between if and the opening parenthesis. You can use astyle as parameterized in utils/indent.sh